centos-7-minimal.ks

```diff
@@ -71,8 +71,8 @@ syslinux
 
 %post
 
-LANG="en_US"
-echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
+# Fix locale issue (same way it happen in boot2docker)
```

Comment:
I would change the comment a bit not referencing boot2docker, but what we actually are doing. Setting a global Locale for the server (assuming that's what it is we do).